airflow/providers_manager.py

```diff
@@ -204,6 +205,11 @@ def wrapped_function(*args, **kwargs):
     return provider_info_cache_decorator
 
 
+MIN_PROVIDER_VERSIONS = {
+    "apache-airflow-providers-celery": "2.1.0",
+}
```

Comment:
Should we maybe put this higher up in the file?

Reply:
Yep. Why not :)